packages/theming/theming-utils/src/isHighContrast.macos.ts

```diff
@@ -0,0 +1,26 @@
+import type { Theme } from '@fluentui-react-native/theme-types';
+
+let isHighContrastEnabled = false;
+
+/**
+ * Used to set isHighContrast state for FURN
+ * Specifically for macOS, where you cannot ask for HC state,
+ * so you have to store the value to keep track of it.
+ *
+ * @platform macOS
+ * @param isHighContrast New value of high contrast state
+ */
```

Comment:
It's occurring to me we could probably just... fix this in React Native macOS so you can ask for the value. Would that simplify things for FURN? @chiuam fyi

Reply:
I mean up to you? I don't think it particularly matters since macOS' implementation would probably be different from RNW anyway..

Reply:
I wasn't sure how bad the workaround in FURN was. It seems like RNW is a custom implementation so converging won't happen anytime soon...

I think more likely we'll push a change to React Native Core for iOS HC, and align to that.

Reply:
re: fixing HC in react-native-macos - Agreed that we should expose an API for detecting HC, I ended up implementing that in FURN as a workaround to unblock ButtonV1.

Reply:
@chiuam with filing a bug? Maybe we can get it with 0.71

Reply:
Oh, cool! I'd be happy to leave that as a followup outside of this PR to be clear, to reduce risk.

Reply:
Sounds good! Maybe when we tap into some of the iOS HC work?